modules/ROOT/pages/logging.adoc

```diff
@@ -202,7 +202,7 @@ On MS Windows `logFilePath` supports the following path formats.
 [source,javascript]
 ----
 "C:/var/tmp/sglogfile.log"
-`C:\var\tmp\sglogfile.log`
+`C:\\var\\tmp\\sglogfile.log`
```

Comment:
A single backslash seems valid looking at https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats

Am I missing something on this page?